### PR TITLE
Fix: Add missing prerequisite for Rewrite URI annotation

### PR DESCRIPTION
**Pull request by d19dotca**
### Description

Added prerequisite to call out that the `RewriteURIs` feature gate needs to be enabled for this annotation to work. 

Fixes an issue discovered when troubleshooting support case #00049247.

### Testing instructions

N/A

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_src/kubernetes-ingress-controller/guides/requests/rewrite-annotation.md [hunk 1]
```diff
@@ -9,7 +9,13 @@ purpose: |
 
 The annotation can be used on `Ingress` and `HTTPRoute` resources, and configures a [request-transformer](/hub/kong-inc/request-transformer/) plugin within Kong when added to a route.
 
-This definition creates a route that matches the path `/users/(\w+)` and rewrites it to `/requests/users_svc/$1` before sending the request upstream.
+This definition creates a route that matches the path `/users/(\w+)` and rewrites it to `/requests/users_svc/$1` before sending the request upstream. 
+
+## Prerequisite
+
+The `RewriteURIs` [feature gate](https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/feature-gates/) must be enabled.
```
Comment (lena-larionova) [inline app/_src/kubernetes-ingress-controller/guides/requests/rewrite-annotation.md:16]:
```suggestion
The `RewriteURIs` [feature gate](/kubernetes-ingress-controller/{{page.release}}/reference/feature-gates/) must be enabled.
```

**Comment by lena-larionova** [comment]
Can you add a `## Troubleshooting` section at the bottom of this same page with the error message that comes up when the feature gate isn't enabled?

**Comment by d19dotca** [comment]
> Can you add a `## Troubleshooting` section at the bottom of this same page with the error message that comes up when the feature gate isn't enabled?

Ohh, I thought it'd be intended to put on the general [Troubleshooting page](https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/troubleshooting/) in a new section there, but if we want to keep this one on the same page I can certainly add that, no worries. :-)

**Comment by lena-larionova** [comment]
@d19dotca you know what - you're right. I'm thinking in dev site terms :) let's stick with the established practice for this change for now and go with your original plan.

**Review by lena-larionova — APPROVED** [review]

**Pull request merged**